src/sentry/preprod/api/models/project_preprod_build_details_models.py

```diff
@@ -37,6 +37,7 @@ class BuildDetailsAppInfo(BaseModel):
     platform: Platform | None = None
     is_installable: bool
     build_configuration: str | None = None
+    app_icon_id: str | None = None
```

Comment:
This is related to app icon, not the generic download endpoint, but should be fine to merge as the frontend does not yet use this until https://github.com/getsentry/sentry/pull/102118 lands